### PR TITLE
CAMEL-17352: Add plain option

### PR DESCRIPTION
**Pull request by Croway**

### CODE REVIEW

**Comment by Croway** [comment]
Seems like camel-log cannot be tested due to cyclic references, like `org.apache.camel:camel-management:3.15.0-SNAPSHOT --> org.apache.camel:camel-log:3.15.0-SNAPSHOT --> org.apache.camel:camel-test-junit5:3.15.0-SNAPSHOT`, is there another way to test it? I'd like to use `CamelTestSupport` features like other components.

**Comment by davsclaus** [comment]
You put tests inside core/camel-core for regular tests, and jmx tests in camel-management

**Comment by Croway** [comment]
> You put tests inside core/camel-core for regular tests, and jmx tests in camel-management

I see, thanks for advice!

**Review by oscerd — APPROVED** [review]

File: components/camel-log/src/main/java/org/apache/camel/component/log/LogEndpoint.java [hunk 1]
```diff
@@ -118,6 +118,8 @@
     @UriParam(label = "formatting", enums = "Default,Tab,Fixed", defaultValue = "Default",
               description = "Sets the outputs style to use.")
     private DefaultExchangeFormatter.OutputStyle style = DefaultExchangeFormatter.OutputStyle.Default;
+    @UriParam(defaultValue = "false", description = "Log exchange body as-is (ex. log(\"${body}\").")
```
Comment (davsclaus) [inline components/camel-log/src/main/java/org/apache/camel/component/log/LogEndpoint.java:121]: Lets use the same description:
```
If enabled only the body will be printed out
```

To make it stand out that all the other options are not considered then

**Review by davsclaus — APPROVED** [review]

**Pull request merged**